### PR TITLE
VACMS-2910: enforce "Rich Text Limited" wysiwyg text format for step paragraph type

### PR DESCRIPTION
**Pull request by acrollet**
## Description

See #2910 

## Testing done

Created & ran behat test.

## QA steps

As a user with permission to create a "step by step" node
1. Create a "step by step" node
1. Confirm that text format selection is not allowed for the "Step" wysiwyg. (In the 'Add Step' section)
1. Then validate Acceptance Criteria from issue
- [x] Text format selection is not allowed for steps
- [x] Available wysiwyg buttons are as expected for "Rich Text Limited" format

## Definition of Done
- [ ] Product release are written (or in progress), if required.
- [ ] Documentation has been updated, if applicable.

### CODE REVIEW

File: tests/behat/features/content_editing.feature [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+  I need to have certain functionality available
+
+@content_editing
+  Scenario: Log in and confirm that "Step by step" node edit has the correct field settings
+    Given I am logged in as a user with the "administrator" role
+    And I am at "node/add/step_by_step"
+    Then I should see "Create Step-by-Step"
+    And I should see "Add Step"
+    # Confirm that text format selection is not allowed for the "Step" paragraph type.
+    And I should not see "Rich Text" in the "#edit-field-steps-0-subform-field-step-0-subform" element
```
Comment (swirtSJW) [inline tests/behat/features/content_editing.feature:14]: Great test.   Can we also assert that we should see  'Rich Text Limited'?

There is a bit of disconnect between ` I need to have certain functionality available` and a test that looks for what should not be there. ;)

Comment (acrollet) [reply tests/behat/features/content_editing.feature:14]: @swirtSJW hmm, excellent point, however Drupal doesn't show us the format in use when only one is available. We could perhaps try to confirm that a few of the appropriate WYSIWYG buttons are available and the text format select element is not present?

Comment (swirtSJW) [reply tests/behat/features/content_editing.feature:14]: oh right... with no choices it presents none.   Ok  Good enough reason.  ;)  Should we also validate plain text is not an option and therefore indirectly and loosely assert that only rich limited is present.

Comment (acrollet) [reply tests/behat/features/content_editing.feature:16]: @swirtSJW updated test pushed. I made a call that trying to look for the wysiwyg buttons with behat is likely to be quite time-consuming - lmk if you think it's worth investing the effort.

**Review by swirtSJW — APPROVED** [review]

**Pull request merged**